identity/src/main/java/com/stripe/android/identity/ml/IDDetectorAnalyzer.kt

```diff
@@ -110,14 +125,19 @@ internal class IDDetectorAnalyzer(modelFile: File) :
         const val OUTPUT_BOUNDING_BOX_TENSOR_INDEX = 0
         const val OUTPUT_CATEGORY_TENSOR_INDEX = 1
         const val OUTPUT_BOUNDING_BOX_TENSOR_SIZE = 4
+        const val INDEX_NO_ID = 0
+        const val INDEX_PASSPORT = 1
+        const val INDEX_ID_FRONT = 2
+        const val INDEX_ID_BACK = 3
+        const val INDEX_INVALID = 4
         val INPUT_TENSOR_TYPE: DataType = DataType.FLOAT32
         val OUTPUT_CATEGORY_TENSOR_SIZE = Category.values().size
         val INDEX_CATEGORY_MAP = mapOf(
-            0 to Category.NO_ID,
-            1 to Category.PASSPORT,
-            2 to Category.ID_FRONT,
-            3 to Category.ID_BACK,
-            4 to Category.INVALID,
+            INDEX_NO_ID to Category.NO_ID,
+            INDEX_PASSPORT to Category.PASSPORT,
+            INDEX_ID_FRONT to Category.ID_FRONT,
+            INDEX_ID_BACK to Category.ID_BACK,
+            INDEX_INVALID to Category.INVALID,
```

Comment:
rather than a map of int -> category, would just a list of category work where index is implicit?

Reply:
these constants are used elsewhere, (e.g when [building a data object](https://github.com/stripe/stripe-android/pull/4724/files#diff-9a8fb3297da21d29a2ec3fd58e4649883ac41c9c2048404a8c95b7c6ea904acb) sent to server), I thought it would be more readable to have explicit names